include/Core/Request/LogListAction.php

```diff
@@ -21,6 +21,7 @@ public function load() {
 		add_action( 'el-log-list-delete', array( $this, 'delete_logs' ) );
 		add_action( 'el-log-list-delete-all', array( $this, 'delete_all_logs' ) );
 		add_action( 'el-log-list-manage-user-roles-changed', array( $this, 'update_capabilities_for_user_roles' ), 10, 2 );
+		add_filter( 'wp_kses_allowed_html', array( $this, 'el_allow_link_tag' ), 10, 2 );
```

Comment:
This will enable `link` tag in all instances where the `wp_kses_post` was used. We don't want that.

Instead, replace `wp_kses_post` with `wp_kses` call and pass the modified tag list to that function.